js/serializers/SCORMSuspendData.js

```diff
@@ -940,13 +940,13 @@ define([
     const esTypes = values.map(value => {
       const valueType = findValueTypeFromValue(value, isFloat);
       if (valueType.esType === 'number') {
-        minValue = _.min([value, minValue]);
-        maxValue = _.max([value, maxValue]);
+        minValue = Math.min([value, minValue]);
+        maxValue = Math.max([value, maxValue]);
```

Comment:
`_.min` - https://underscorejs.org/#min
Takes an array or values as the first argument.

`Math.min` - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min
Takes values as the arguments.

Reply:
could maybe make use of the Math.min shallow copying of the array with `Math.min(...array1)`, so in this case it could be `maxValue = Math.max(...[value, maxValue]);` what do you think?

Reply:
```suggestion
        minValue = Math.min(value, minValue);
        maxValue = Math.max(value, maxValue);
```
This should suffice? Just removing the wrapping array.